File: src/test/java/com/intellij/plugin/applescript/test/parsing/TellApplicationMusicTest.java
package com.intellij.plugin.applescript.test.parsing;

import com.intellij.testFramework.fixtures.BasePlatformTestCase;

public class TellApplicationMusicTest extends BasePlatformTestCase {

    public void testTellApplicationMusic() {
        String script = "tell application \"Music\"\n" +
                        "  property effect\n" +
                        "  get properties\n" +
                        "end tell";

        myFixture.configureByText("test.applescript", script);
        myFixture.checkHighlighting(false, false, true, false);
    }
}

File: src/test/kotlin/com/intellij/plugin/applescript/test/parsing/TellApplicationMusicTest.kt
package com.intellij.plugin.applescript.test.parsing

import com.intellij.testFramework.fixtures.BasePlatformTestCase

class TellApplicationMusicTest : BasePlatformTestCase() {
    fun testTellApplicationMusic() {
        val script =
            """
            tell application "Music"
              property effect
              get properties
            end tell
            """.trimIndent()

        myFixture.configureByText("test.applescript", script)
        myFixture.checkHighlighting(false, false, true, false)
    }
}
